go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/filecoin-project/go-data-transfer v1.15.2
 	github.com/filecoin-project/go-state-types v0.1.0
-	github.com/filecoin-project/storetheindex v0.5.1
+	github.com/filecoin-project/storetheindex v0.5.2-0.20221129130444-841de0aa8b23
```

Comment:
tag sti and merge against a released STI version, ideally

Reply:
Will do that after approval of: https://github.com/filecoin-project/storetheindex/pull/1081